### DISCUSSION TITLE
ComfyUI x Refiners

### DISCUSSION THREAD

**Discussion by deltheil** [finegrain-ai/refiners#430] [Announcements]
# We've created some custom ComfyUI nodes! They are published under the [comfyui-refiners](https://registry.comfy.org/publishers/finegrain/nodes/comfyui-refiners) registry

We only support our new [Box Segmenter](https://huggingface.co/spaces/finegrain/finegrain-object-cutter) at the moment, but we're thinking of adding more nodes since there seems to be a demand for it.

We leverage the new (beta) [Comfy Registry](https://registry.comfy.org/) to host our nodes. They are available at: https://registry.comfy.org/publishers/finegrain/nodes/comfyui-refiners.

You can install them by doing:

```bash
comfy node registry-install comfyui-refiners
```

Or by unzipping the archive you can download by clicking "Download Latest" into your custom_nodes comfy folder.

# End-to-end workflow

We also provide an [example workflow](https://github.com/finegrain-ai/refiners/blob/main/src/comfyui-refiners/assets/box_segmenter.json) leveraging GroundingDINO and our [BoxSegmenter](https://huggingface.co/finegrain/finegrain-box-segmenter) to extract objects from an image:

![box_segmenter](https://raw.githubusercontent.com/finegrain-ai/refiners/main/src/comfyui-refiners/assets/box_segmenter.png)

# 💬 Share Your Feedback & Ideas!

We are eager to hear your feedback and suggestions for new nodes and how you'll use them! 🙏

**Comment by martintomov** [comment]
Loving it! Hoping to see your object-eraser model there soon as well.

**Comment by deltheil** [comment]
# [Update] Check out the updated installation guide here: https://refine.rs/guides/comfyui_refiners/

cc @martintomov @515753541 (@Laurent2916)
